internal/corazarules/rule_match.go

```diff
@@ -130,21 +130,19 @@ func (mr MatchedRule) details(matchData types.MatchData) string {
 
 func (mr MatchedRule) matchData(matchData types.MatchData) string {
```

Comment:
I know it's a bit unrelated, but while we're here, can we have this function accept `log *strings.Builder` and pass it in from the caller instead of creating a new one?